src/scripts/embed/server/views/video.handlebars

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>{{title}}</title>
+  <link rel="stylesheet" href="/embed.css">
+  <script src="/bundle.js"></script>
```

Comment:
This `bundle.js` is not the same one used by the portal. Rather, it is `build/embed/bundle.js`, which is only concerned w/ dependencies required to render our embeddable player.